operator/src/main/java/com/instaclustr/cassandra/operator/K8sVersionValidator.java

```diff
@@ -37,7 +37,7 @@ public Void call() throws Exception {
 
         logger.debug("Server version: {}", versionInfo);
 
-        if (!options.noVersionCheck) {
+        if (options.noVersionCheck) {
```

Comment:
Not sure what's happening here, but flag doesn't appear to work. 
java -jar /Users/ben/git/cassandra-operator/docker/cassandra-operator/operator-1.0-SNAPSHOT-final.jar --no-version-check with minikube (which on v0.25.2 with 1.9.2 provides no minor or major string) fails to skip the version check

Reply:
Right, because the `Options` object isn't injected, so it'll always be false. Oops.